test/aria/widgets/container/dialog/modalMaskSize/DialogModalMaskSize.tpl

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2017 Amadeus s.a.s.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+{Template {
+    $classpath:"test.aria.widgets.container.dialog.modalMaskSize.DialogModalMaskSize"
+}}
+
+    {macro main()}
+        {@aria:Button {
+            label: "Refresh",
+            onclick: {
+                fn: this.$refresh,
+                scope: this,
+                resIndex: -1
+            }
+        } /}
+        {@aria:Dialog {
+            modal: true,
+            movable: true,
+            maximizable: true,
+            resizable: true,
+            macro: "dialogContent",
+            bind: {
+                xpos: {
+                    inside: data,
+                    to: "x"
+                },
+                ypos: {
+                    inside: data,
+                    to: "y"
+                },
+                width: {
+                    to: "width",
+                    inside: data
+                },
+                height: {
+                    to: "height",
+                    inside: data
+                },
+                center: {
+                    to: "center",
+                    inside: data
+                },
+                visible: {
+                    to: "visible",
+                    inside: data
+                }
+            }
+        }/}
+        {@aria:CheckBox {
+            label: "Dialog visible",
+            bind: {
+                value: {
+                    to: "visible",
+                    inside: data
+                }
+            }
+        } /}
+        <div style="width: 100px; height: 2000px; background-color:blue;"></div>
+        <input {id "myInput"/}>
+        {@aria:CheckBox {
```

Comment:
You have an identical Checkbox just above. Is it a mistake or done on purpose?

Reply:
It is done on purpose to be able to trigger the display of the dialog whether the page is scrolled or not (there is a big div separating both checkboxes).